.github/workflows/build_linux_packages.yml

```diff
@@ -136,14 +142,30 @@ jobs:
 
       # TODO: Move to script
       - name: Upload Logs
+        if: always()
```

Comment:
It's time to move this logic to a Python script.

Reply:
so you want me write this in Python as a separated file AFAIU. Where should I locate it?

Reply:
Pick somewhere under https://github.com/ROCm/TheRock/tree/main/build_tools. Maybe `upload_logs_to_s3.py`